lib/ddtrace/contrib/mongodb/subscribers.rb

```diff
@@ -11,15 +11,17 @@ module MongoDB
       # system available in the Mongo driver.
       class MongoCommandSubscriber
         def started(event)
-          pin = Datadog::Pin.get_from(event.address)
-          return unless pin && pin.enabled?
+          service = Datadog::Tracing.configuration_for(event.address, :service_name) \
+                    || Datadog::Tracing.configuration[:mongo, event.address.seed][:service_name]
+
+          return unless Datadog::Tracing.enabled?
```

Comment:
Minor: Should the `enabled?` check be moved earlier? We'd save all of the work of looking up the documentation (and thus have less of an impact on applications where tracing is disabled).

Reply:
Makes sense.